internal/reader/home_chain.go

```diff
@@ -348,25 +366,43 @@ type ChainConfig struct {
 	Config chainconfig.ChainConfig `json:"config"`
 }
 
-// OCR3Config mirrors CCIPConfig.sol's OCR3Config struct
+// nolint: lll
+// https://github.com/smartcontractkit/chainlink/blob/e964798a974f3246ee1da011feffe33509b358df/contracts/src/v0.8/ccip/capability/CCIPHome.sol#L105-L131
+
+type OCR3Node struct {
+	P2pID          [32]byte `json:"p2pId"`
+	SignerKey      []byte   `json:"signerKey"`
+	TransmitterKey []byte   `json:"transmitterKey"`
+}
+
+// nolint: lll
+// https://github.com/smartcontractkit/chainlink/blob/e964798a974f3246ee1da011feffe33509b358df/contracts/src/v0.8/ccip/capability/CCIPHome.sol#L105-L131
+
 type OCR3Config struct {
 	PluginType            uint8                   `json:"pluginType"`
 	ChainSelector         cciptypes.ChainSelector `json:"chainSelector"`
-	F                     uint8                   `json:"F"`
+	FRoleDON              uint8                   `json:"fRoleDON"`
 	OffchainConfigVersion uint64                  `json:"offchainConfigVersion"`
 	OfframpAddress        []byte                  `json:"offrampAddress"`
 	RmnHomeAddress        []byte                  `json:"rmnHomeAddress"`
-	P2PIds                [][32]byte              `json:"p2pIds"`
-	Signers               [][]byte                `json:"signers"`
-	Transmitters          [][]byte                `json:"transmitters"`
+	Nodes                 []OCR3Node              `json:"nodes"`
 	OffchainConfig        []byte                  `json:"offchainConfig"`
 }
 
-// OCR3ConfigWithmeta mirrors CCIPConfig.sol's OCR3ConfigWithMeta struct
+// OCR3ConfigWithMeta
+// nolint: lll
```

Comment:
```suggestion
//nolint:lll
```

Reply:
This isn't followed in so many places in our codebase :( Will fix these occurrences